docs/src/pages.js

```diff
@@ -189,6 +189,7 @@ const pages = [
       { pathname: '/system/screen-readers' },
       { pathname: '/system/typography' },
       { pathname: '/system/properties' },
+      { pathname: '/system/advanced' },
```

Comment:
I wonder if it wouldn't be better to have this structure:

- "Basics"
- "Advanced"
- "Properties"
- x
- y

Reply:
- "Basics"
- "Properties"
- x
- y
- "Advanced"

?